wodles/aws/aws_s3.py

```diff
@@ -1006,22 +1004,32 @@ def _event_should_be_skipped(event_):
 
                 self.send_event(event_msg)
 
+    @staticmethod
+    def _print_not_logs_to_prcess_message(bucket, aws_account_id, aws_region):
```

Comment:
Typo in the method's name
```suggestion
    def _print_no_logs_to_process_message(bucket, aws_account_id, aws_region):
```